bug: Investigate flaky `TestStore_InsertCleansUpOldUptimeEntriesProperly` test

### Describe the bug

Sometimes, the `TestStore_InsertCleansUpOldUptimeEntriesProperly` test will fail

### What do you see?

https://github.com/TwiN/gatus/actions/runs/3723336201/attempts/1
```
 --- FAIL: TestStore_InsertCleansUpOldUptimeEntriesProperly (0.04s)
    sql_test.go:109: oldest endpoint uptime entry should've been ~5 hours old, was 4h59m40.064240509s
    sql_test.go:119: oldest endpoint uptime entry should've been ~5 hours old, was 4h59m40.068831372s
    sql_test.go:129: oldest endpoint uptime entry should've been ~8 hours old, was 7h59m40.073425836s
    sql_test.go:139: oldest endpoint uptime entry should've been ~239h0m0s hours old, was 238h59m40.0780609s
    sql_test.go:150: oldest endpoint uptime entry should've been ~8 hours old, was 7h59m40.08307097s
```

### What do you expect to see?

Test should pass

### List the steps that must be taken to reproduce this issue

1. Execute `TestStore_InsertCleansUpOldUptimeEntriesProperly` between `hh:59:30` and `hh:59:59`
2. Note how the test will fail

### Version

latest (master)

### Additional information

I suspect this has something to do with https://github.com/TwiN/gatus/blob/f6a621da285dd43b3207ec21e4101b4b11eff6c5/storage/store/sql/sql_test.go#L100

--- a/storage/store/sql/sql_test.go
+++ b/storage/store/sql/sql_test.go
@@ -97,7 +97,7 @@ func TestNewStore(t *testing.T) {
 func TestStore_InsertCleansUpOldUptimeEntriesProperly(t *testing.T) {
 	store, _ := NewStore("sqlite", t.TempDir()+"/TestStore_InsertCleansUpOldUptimeEntriesProperly.db", false)
 	defer store.Close()
-	now := time.Now().Round(time.Minute)
+	now := time.Now().Truncate(time.Hour)
 	now = time.Date(now.Year(), now.Month(), now.Day(), now.Hour(), 0, 0, 0, now.Location())
 
 	store.Insert(&testEndpoint, &core.Result{Timestamp: now.Add(-5 * time.Hour), Success: true})

